data/lib/src/network/linshare_http_client.dart

```diff
@@ -691,7 +691,7 @@ class LinShareHttpClient {
             .withPathParameter(uploadRequestGroupId.uuid)
             .withPathParameter(Endpoint.recipents)
             .generateEndpointPath(),
-        data: recipients.map((data) => GenericUserDto(data.mail, lastName: data.lastName, firstName: data.firstName).toJson()).toList().toString());
```

Comment:
you can ignore fields in `toJson` by annotation, no need to create another extensions